jujugui/static/gui/src/app/store/env/sandbox.js

```diff
@@ -1517,6 +1518,23 @@ YUI.add('juju-env-sandbox', function(Y) {
     },
 
     /**
+      Handle KeyManager.ListKeys messages.
+
+      @method handleKeyManagerListKeys
+      @param {Object} data The contents of the API arguments.
+      @param {Object} client The active ClientConnection.
+      @param {Object} state An instance of FakeBackend.
+    */
+    handleKeyManagerListKeys: function(data, client, state) {
+      client.receive({
+        'request-id': data['request-id'],
+        response: {results: [{
+          result: ['sandbox SSH key']
```

Comment:
should this handle the fullkey/fingerprint difference so that it can be replicated a bit in testing and such?